eland/ml/transformers/__init__.py

```diff
@@ -93,5 +93,5 @@ def get_model_transformer(model: Any, **kwargs: Any) -> ModelTransformer:
         "LGBMClassifierTransformer",
     ]
     _MODEL_TRANSFORMERS.update(_LIGHTGBM_MODEL_TRANSFORMERS)
-except ImportError:
+except (ImportError, OSError):
```

Comment:
Since the OSError isn't an exception one normally receives from an import can we explain the situation we're defending against in a comment nearby?